### PR TITLE
Cockpit 175

### PR DESCRIPTION
**Pull request by mvollmer**
https://mvollmer.github.io/cockpit-project.github.io/blog/cockpit-175.html

### CODE REVIEW

File: _posts/2018-08-08-cockpit-175.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We
+release regularly.  Here are the release notes from version 175.
+
+### Network bound disk encryption
+
+It is now possible to manage the key slots of LUKSv1 encrypted
+devices. In addition to traditional passphrases, key slots can also be
+bound to a [Tang](https://github.com/latchset/tang) server via
+[Clevis](https://github.com/latchset/clevis).
```
Comment (garrett) [inline _posts/2018-08-08-cockpit-175.md]: Suggested mod:

> This release adds encrypted storage support for passphrases and network keyservers.
>
> Under the hood, LUKSv1 manages encrypted devices, allowing up to 8 different key "slots". Each slot may contain a passphrase or can be bound to a [Tang](https://github.com/latchset/tang) server using [Clevis](https://github.com/latchset/clevis).
>
> In a future release, we're planning to add key groups and more authentication types. Key groups bypass the 8 slot limitation and provide complex authentication rules. Stay tuned!

File: _posts/2018-08-08-cockpit-175.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+<iframe width="960" height="720" src="https://www.youtube.com/embed/GAPJVN6EEz4?rel=0" frameborder="0" allowfullscreen></iframe>
```
Comment (garrett) [inline _posts/2018-08-08-cockpit-175.md]: If you'd like, I can clean up the audio on the video. Just give me the original file (somehow) and I'll process it.

File: _posts/2018-08-08-cockpit-175.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+slug: cockpit-175
+category: release
+summary: Network bound disk encryption
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We
+release regularly.  Here are the release notes from version 175.
+
+### Network bound disk encryption
```
Comment (allisonkarlitskaya) [inline _posts/2018-08-08-cockpit-175.md]: The section heading "Network bound disk encryption" does not accurately describe the contents of the section.  NBDE is only one subfeature of the ability to manage LUKS

Comment (garrett) [reply _posts/2018-08-08-cockpit-175.md]: Excellent point!

Perhaps "Storage: Key management for encrypted devices" might be a better fit?

File: _posts/2018-08-08-cockpit-175.md (hunk 1 above)
Comment (mvollmer) [reply _posts/2018-08-08-cockpit-175.md]: Very good, thanks!

File: _posts/2018-08-08-cockpit-175.md (hunk 2 above)
Comment (mvollmer) [reply _posts/2018-08-08-cockpit-175.md]: It#s just for the release notes, I think it's fine.  I should ask for a proper mike.

File: _posts/2018-08-08-cockpit-175.md (hunk 3 above)
Comment (mvollmer) [reply _posts/2018-08-08-cockpit-175.md]: Very good!

File: _posts/2018-08-08-cockpit-175.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+### Summary: Key management for encrypted devices
+
+This release adds encrypted storage support for passphrases and
+network keyservers.
+
+Under the hood, LUKSv1 manages encrypted devices, allowing up to 8
+different key "slots". Each slot may contain a passphrase or can be
+bound to a [Tang](https://github.com/latchset/tang) server using
+[Clevis](https://github.com/latchset/clevis).
```
[the remark this answers is not in the document]
Comment (mvollmer) [reply _posts/2018-08-08-cockpit-175.md]: Done

File: _posts/2018-08-08-cockpit-175.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+
+### Try it out
+
+Cockpit 175 is available now:
+
+ * [For your Linux system](https://cockpit-project.org/running.html)
+ * [Source Tarball](https://github.com/cockpit-project/cockpit/releases/tag/175)
+ * [Fedora 28](https://bodhi.fedoraproject.org/updates/cockpit-175-1.fc28)
+
+*[VM]: Virtual Machine
```
[the remark this answers is not in the document]
Comment (mvollmer) [reply _posts/2018-08-08-cockpit-175.md]: Done

**Comment by garrett** [comment]
LGTM :+1:

**Pull request merged**